examples/react/nextjs-app-prefetching/app/get-query-client.ts

```diff
@@ -0,0 +1,33 @@
+import { QueryClient, defaultShouldDehydrateQuery } from '@tanstack/react-query'
+
+function makeQueryClient() {
+  return new QueryClient({
+    defaultOptions: {
+      queries: {
+        staleTime: 60 * 1000,
+      },
+      dehydrate: {
```

Comment:
Not sure if it's just me, but I'm getting a TS error here:

```Object literal may only specify known properties, and 'dehydrate' does not exist in type 'DefaultOptions<Error>'.```

Reply:
you need to build the `query-core` package locally to make this type correctly:

```
npx nx run @tanstack/query-core:build
```

Reply:
Of course I do.. 🤦 Thanks, that solved it of course.